### ISSUE TITLE
Fix tests in bootstrap-5 branch

### ISSUE THREAD

**Issue by thimo** [bootstrap-ruby/bootstrap_form#582]
Currently 8 tests fail in the bootstrap-5 branch: 
```
Finished in 1.009611s, 236.7248 runs/s, 274.3631 assertions/s.
239 runs, 277 assertions, 8 failures, 0 errors, 0 skips
```

I'll try to fix these.

**Comment by thimo** [comment]
PR has been created: https://github.com/bootstrap-ruby/bootstrap_form/pull/583

**Comment by lcreid** [comment]
Fixed by #583
